src/lib/transition_frontier/transition_frontier.ml

```diff
@@ -576,13 +579,16 @@ struct
     Hashtbl.remove t.table t.root ;
     Hashtbl.set t.table ~key:new_root_hash ~data:new_root_node ;
     t.root <- new_root_hash ;
+    (* TODO: these metrics are too expensive to compute in this way, but it should be ok for beta *)
+    (*
     let num_finalized_staged_txns =
       Breadcrumb.user_commands new_root |> List.length |> Float.of_int
     in
```

Comment:
Can you explain a little bit why this is expensive?

Reply:
I guess it's because we are using `List.length` and `List.@`...

Reply:
But it shouldn't be that expensive to compute `num_finalized_staged_txns`.

Reply:
You're probably correct that this one may not be that expensive. I should just add this one back in. The comment is in reference to `Ledger.Db.to_list t.root_snarked_ledger`, which is expensive and needed to be commented out.